apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -115,25 +119,34 @@ export default function RowColumn({ index, style, data, column = false }) {
     setClassArr(clazzArr);
   }, [cell && cell.qState]);
 
-  const getCheckboxField = ({ lbl, highlighted, color, qElemNumber }) => {
-    const cb = <ListBoxCheckbox label={lbl} highlighted={highlighted} checked={isSelected} />;
+  const getCheckboxField = ({ lbl, color, qElemNumber }) => {
+    const cb = <ListBoxCheckbox label={lbl} checked={isSelected} />;
+    const labelTag =
+      typeof lbl === 'string' ? (
+        <Typography component="span" noWrap>
+          {lbl}
+        </Typography>
+      ) : (
+        lbl
+      );
     return (
       <FormControlLabel
         color={color}
         control={cb}
-        className={classes.checkboxLabel}
-        label={<Typography>{lbl}</Typography>}
+        className={[classes.checkboxLabel].join(' ').trim()}
```

Comment:
why there should be no to create an array with a single element and then join and trim it